packages/core/src/components/ozon-content/nodes/table.node/colspec/colspec-mapper.ts

```diff
@@ -7,12 +7,14 @@ export function mapColspecs(count: number, nodeList: NodeListOf<Element>): Colsp
   return {
     totalWidth,
     count,
-    columns: elements.map((element, index) => {
+    columns: elements.map((element, index): Colspec => {
       const colNumber = element.getAttribute("colnum");
 
       return {
+        colsep: element.getAttribute("colsep"),
```

Comment:
Hier moeten we maandag even over brainen, ik begrijp niet waarom ik zo vaak `getAttribute("colsep")` tegenkom: Het lijkt mij dat er een data mapping is, vanaf waar we verdere beslissingen nemen.

Maar nu lijkt het alsof we beslissingen nemen op gemapte data, om vervolgens terug te gaan naar de originele data.

Reply:
Lijkt me goed, want ik snap niet wat je bedoelt.

Reply:
Ik heb in het issue beschreven hoe de overerving van colsep en rowsep werkt.

Reply:
Zie volgende opmerking